aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -699,6 +699,30 @@ Tensor sub_zerotensor(const Tensor& self, const Tensor& other, const Scalar& alp
   return maybe_add_maybe_sub(self, other, -alpha);
 }
 
+Tensor linalg_cross_zerotensor(
+  const Tensor& input,
+  const Tensor& other,
+  const int64_t dim)
+{
+  auto out_device = correct_out_device(input, other);
+  // hack to use the TensorIterator to get the correct broadcasting and type
+  // promotion logic (see add_zerotensor)
+  auto device = Device(DeviceType::Meta);
+  auto meta_out = at::redispatch::linalg_cross(
+    c10::DispatchKeySet(at::DispatchKey::Meta),
+    input.to(device),
+    other.to(device),
+    dim);
+
+  if (input._is_zerotensor() || other._is_zerotensor()) {
```

Comment:
but this condition will always be true, right? we only call `linalg_cross_zerotensor` when the `ZeroTensor` dispatch key is set on one of the inputs